src/pages/RedirectFromOAuth.js

```diff
@@ -1,16 +1,17 @@
 import { UserManager } from 'oidc-client-ts'
-import { div, h } from 'react-hyperscript-helpers'
-import { centeredSpinner } from 'src/components/icons'
-import { getOidcConfig } from 'src/libs/auth'
+import { div } from 'react-hyperscript-helpers'
 import { useOnMount } from 'src/libs/react-utils'
 
 
-const RedirectFromOAuth = silent => {
-  const userManager = new UserManager(getOidcConfig())
+const RedirectFromOAuth = (silent = false) => {
+  const userManager = new UserManager({
+    popup_redirect_uri: `${window.origin}/redirect-from-oauth`,
+    silent_redirect_uri: `${window.origin}/redirect-from-oauth-silent`
+  })
   const url = window.location.href.replace('#', '')
   useOnMount(() => silent === true ? userManager.signinSilentCallback(url) : userManager.signinPopupCallback(url))
   return div({ role: 'main' }, [
-    h(centeredSpinner, { style: { position: 'fixed' } })
+    'Signing in to Terra...'
```

Comment:
The `centeredSpinner` icon requires the `config.json` to be loaded (to get the colors for the branded site…). By just having text I can avoid loading `config.json` in the auth pop-up. I figure this is fine because it's visible for a fraction of a second..